dependencies.yaml

```diff
@@ -95,14 +95,12 @@ test_base_requires:
   perl(Test::Warnings): '>= 0.029'
   qemu: '>= 4.0'
   qemu-tools:
-  qemu-x86:
   procps:
   python3-setuptools:
   xterm-console:
   xorg-x11-Xvnc:
   icewm:
   xterm:
-  ipxe-bootimgs:
```

Comment:
You still need to commit an updated `Dockerfile` it seems:

```
9: 
9: #   Failed test 'No changed files after 'make update-deps''
9: #   at xt/27-make-update-deps.t line 25.
9: #  M container/os-autoinst_dev/Dockerfile
9: # Looks like you failed 1 test of 2.
9: xt/27-make-update-deps.t .. 
```

Reply:
Right. We have `ifarch x86_x86` in `os-autoinst.spec` which might be easier than supporting that in `dependencies.yaml`.

Reply:
@Martchus I understood how I can use a similar approach as for "ocr", adapted now

@kalikiana ifarch x86_64 is not the same as ifnarch s390x